mobile/build.gradle

```diff
@@ -170,10 +169,16 @@ dependencies {
 
     testCompile project(':testlib')
     testCompile 'org.mockito:mockito-core:1.10.8'
+    compile('com.crashlytics.sdk.android:crashlytics:2.2.0@aar') {
+        transitive = true;
```

Comment:
I'm fairly positive transitives are automatically included if you just do the single line. I'd prefer that, even if' it's not what their docs say